### DISCUSSION TITLE
Meta data

### DISCUSSION THREAD

**Discussion by Hendrik1987** [DHI/modelskill#22] [Ideas]
From first examples I see mr.add_observation(…, name = …). Keeping track of meta data like name, frequency band, location and many more is a time consuming task and prone to errors. It would be great to offer a smart way to link files and meta data. Maybe a relational database approach, where files and meta data are linked in lookup tables. Or as part of the files, but that would require to shift to netCDF for example, and only shift the task of relating the files and meta data to the per-processing step. Something to think about...

**Comment by Hendrik1987** [comment]
I suggest additional flexibility by adding an attrs dict to observations and model results, similar to xarray's attrs http://xarray.pydata.org/en/stable/generated/xarray.DataArray.attrs.html
Also relevant: xarray's logic for plot labels http://xarray.pydata.org/en/stable/plotting.html#simple-example

**Comment by jsmariegaard** [comment]
I still like the attrs dict which could also be used for filtering data.
